com.github.hugolabe.Wike.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app-id" : "com.github.hugolabe.Wike",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.38",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "wike",
+  "finish-args" : [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio"
+  ],
+  "build-options" : {
+    "cflags" : "-O2 -g",
+    "cxxflags" : "-O2 -g",
+    "env" : {
+      "V" : "1"
+    }
+  },
+  "cleanup" : [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules" : [
+  	"python3-requests.json",
+    {
+      "name" : "wike",
+      "buildsystem" : "meson",
+      "config-opts" : [
+        "--libdir=lib"
+      ],
+      "builddir" : true,
+      "sources" : [
+        {
+          "type" : "git",
+          "url" : "https://github.com/hugolabe/Wike.git"
```

Comment:
should be a tarball or a tagged commit

Reply:
Thanks! It's my first submit. I think it's correct now...